package.json

```diff
@@ -35,7 +35,8 @@
 		"node-watch": "^0.7.1",
 		"open": "^8.4.0",
 		"ora": "^3.4.0",
-		"p-limit": "^4.0.0"
+		"p-limit": "^4.0.0",
+		"simple-zstd": "^1.4.2"
```

Comment:
In particular, can this be added without needing to install `zstd` separately? I believe `curl` requires `zstd` so it should already be installed most of the time. If it's needed, where do I update the docs?

Reply:
Unfortunately no, zstd is dependant and needs to be installed per operating system. We'd be good to defer to the library itself (from FB) to give instuctions.